### PR TITLE
Added Recipe ContextType with tests

### PR DESCRIPTION
**Pull request by hollodk**
I have added the Recipe ContextType if you like that.

### CODE REVIEW

File: src/ContextTypes/Recipe.php [hunk 1]
```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace JsonLd\ContextTypes;
+
+class Recipe extends CreativeWork
+{
+    /**
+     * Property structure
+     * reference: https://schema.org/Recipe (alphabetical order)
```
Comment (boly38) [inline src/ContextTypes/Recipe.php:9]: would be great to respect alphabetical order or remove this tips ;)

File: src/ContextTypes/Recipe.php [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+     *
+     * @var array
+     */
+    protected $structure = [
+        'name' => null,
```
Comment (boly38) [inline src/ContextTypes/Recipe.php:14]: you must not repeat an entry that already exists in parents (Creative Work or Thing).

File: src/ContextTypes/Recipe.php [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+        'prepTime' => null,
```
Comment (boly38) [inline src/ContextTypes/Recipe.php:15]: some attributes seems to be related to an intermediate ContextType called 'HowTo'

Thing > Creative Work > HowTo > Recipe

cf. https://schema.org/HowTo

Maybe move theses attributes into this new contextType is better for future improvements / addition.

File: src/ContextTypes/Recipe.php [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+        'cookTime' => null,
+        'totalTime' => null,
```
Comment (boly38) [inline src/ContextTypes/Recipe.php:17]: HowTo

File: src/ContextTypes/Recipe.php [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+        'image' => null,
```
Comment (boly38) [inline src/ContextTypes/Recipe.php:18]: already in 'Thing' parent context type

etc..

File: tests/ContextTypes/RecipeTest.php [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+                'name' => 'second review',
+                'reviewRating' => 5
+            ],
+        ],
+    ];
+
+    /**
+     * @test
+     */
+    public function shouldHaveUrl()
```
Comment (boly38) [inline tests/ContextTypes/RecipeTest.php:66]: change method name because you are expecting a recipe not an url

**Review by boly38 — CHANGES_REQUESTED** [review]
Thanks for the work. Need some minor changes for me 👍😃

File: src/ContextTypes/NutritionInformation.php [hunk 7]
```diff
@@ -0,0 +1,20 @@
+<?php
+namespace JsonLd\ContextTypes;
+
+class NutritionInformation extends AbstractContext
```
Comment (boly38) [inline src/ContextTypes/NutritionInformation.php:4]: seems a child of'Thing' ?
https://schema.org/NutritionInformation

and test? :p

**Comment by Torann** [comment]
Recipe was added in version 0.0.19

**Pull request closed without merging**